### ISSUE TITLE
A question about accessibility testing with TalkBack

### ISSUE THREAD

**Issue by khsbory** [google/Accessibility-Test-Framework-for-Android#23]
Hello.
My name is Hyongsop Kim from Korea and my main job is Accessibility Testing.
Originally, the question should be left in the TalkBack GitHub repository, but there is no menu for registering issues in the repository, so I was forced to leave a question here. Please excuse me.
As you know, the updated version 8.1 source code of TalkBack was released in February this year.
But when I downloaded the released version of the source code, I found out that the sample demo app for testing TalkBack has disappeared.
The app provided various screens to test how TalkBack responds in each view or method, and I am so sorry that the app is missing.
So is there any way I can download the source code of the app again?
I often have to offer Android developers a solution to improve accessibility.
If I suggest the method and show the developers a demo app that TalkBack responds to the method, it will give them a little more understanding of accessibility.
So I really need a demo app. Please reply.
